challenge-08/challenge-08.js

```diff
@@ -48,28 +54,65 @@ vai efetuar. Será uma string com os valores `+`, `-`, `*`, `/` ou `%`;
   - Se o operador não for válido, retornar a frase:
   "Operação inválida."
 */
-// ?
+function calculator ( operationName ){
+	return function( numberOne, numberTwo ){
+		var result;
+		
+		switch ( operationName ) {
+
+			case '+':
+				result = numberOne + numberTwo;
+				break;
+
+			case '-':
+				result = numberOne - numberTwo;
+				break;
+
+			case '*':
+				result = numberOne * numberTwo;
+				break;
+
+			case '/':
+				result = numberOne / numberTwo;
+				break;
+
+			case '%':
+				result = numberOne % numberTwo;
+				break;
+
+			default:
+				result = 'Operação inválida';
```

Comment:
-1
Se você colocar `Operação Inválida` na variável result, na hora de retornar será mostrada toda a mensagem:

```
Resultado da operação: 1 LALA 2 = Operação Inválida
```

Isso se for passado `LALA` como operador. Aqui você pode retornar direto essa frase, para nem mostrar a outra de "Resultado da operação".